content/applications/sales/crm/track_leads/quality_leads_report.rst

```diff
@@ -0,0 +1,205 @@
+   :align: center
+   :alt: The Add Custom Filter pop-up with emphasis on the Include Archived toggle.
+
+Enabling this feature adds archived (inactive) leads to the report.
+
+.. _quality_leads_report/add-quality-rules:
+
+Add rules for quality leads
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This step is optional, based on how an organization defines a *quality lead*. For more information,
+see :ref:`Define quality leads <track_links/define-a-lead>`.
+
+View the report
+===============
+
+.. important::
+   At the top of the :guilabel:`Add Custom Filter` form, there is an option to match :guilabel:`any`
+   or :guilabel:`all` of the rules. In order to properly run the report, only records that match
+   **all** of the following filters should be included. Before adding the filters, make sure
+   :guilabel:`all` is selected in this field.
+
+   .. image:: quality_leads_report/match-all-rules.png
+     :align: center
+     :alt: Close up on the match all rules option on the add a custom filter pop-up window.
+
+After the filters are configured, click :guilabel:`Add`. The default display for the report is a bar
+graph, where the leads are grouped by *stage*.
+
+To group the results by salesperson, click the :guilabel:`🔻(triangle pointed down)` icon to the
+right of the :guilabel:`Search...` bar to open the drop-down menu. Under the :guilabel:`Group By`
+heading, select :guilabel:`Salesperson`. Under the same :guilabel:`Group By` heading, click
+:guilabel:`Add a Custom Group`, then select :guilabel:`Active` on the resulting drop-down menu.
+
+The report now displays the total count of *quality leads* each salesperson has received in the
+designated time period. The leads are color-coded to identify whether they are *active* or *marked
+as lost*.
+
+.. tip::
+   To save this search for later, click the :guilabel:`🔻(triangle pointed down)` icon next to the
+   :guilabel:`Search...` bar to open the drop-down menu. Under the :guilabel:`Favorites` heading,
+   click :guilabel:`Save current search`.
+
+   In the drop-down menu, rename the report from `Pipeline Analysis` to `Quality Leads`, and click
+   :guilabel:`Save`.
+
+.. _track_links/define-a-lead:
+
+Define a quality lead
+=====================
+
+A *quality lead* is one that is likely to result in a sale. Although the exact criteria for a
+quality lead varies from organization to organization, it is often a combination of factors
+commonly believed to be the most beneficial to a salesperson, in addition to factors valued by the
+specific organization.
+
+In addition to the basic filters and grouping options outlined in the general :ref:`Quality leads
+report <track_links/create-quality-leads-report>`, consider the following filters when defining a
+quality lead:
+
+- :guilabel:`Email` or :guilabel:`Phone`: the information in these fields can help determine whether
+  or not a lead is a professional contact.
+- :guilabel:`Source`: this field links to the marketing and lead generation efforts from other Odoo
```

Comment:
I like adding campaign, because even if it is more about tracking the success of the campaign to capture quality leads, it is still useful to see what else you can do with the filters and the report itself.